src/components/Auth/ValidationMessageComponent/ValidationMessageComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import { Text } from "react-native";
+import React, { Fragment, FunctionComponent } from "react";
+import uuid from "react-native-uuid";
+import { PasswordValidationPropmtMessage } from "../hooks/usePasswordValidation";
+import { useTheme } from "../../../hooks/theme-context";
+
+const ValidationMessageComponent: FunctionComponent<{
```

Comment:
If the idea is to make a generic nice component to show error messages the argument should not require the `PasswordValidationPropmtMessage[]`.

Since the PasswordValidationPropmtMessage just contain error: string. Can you declare a generic new type which makes more sense? You can extend and implement types if you want.

Reply:
I have renamed the type and moved it to the ValidationMessageComponent.